frontend/lib/src/components/widgets/DataFrame/hooks/useColumnLoader.ts

```diff
@@ -215,17 +240,22 @@ export function getColumnType(column: BaseColumnProps): ColumnCreator {
  * @param data - The Arrow data extracted from the proto message
  * @param disabled - Whether the widget is disabled
  *
```

Comment:
nit: `@param columnOrder` is missing. An explanation of what `string[]` is supposed to be would be great as well

Reply:
Updated 👍